internal/pkg/cli/interfaces.go

```diff
@@ -241,8 +242,8 @@ type wsPipelineManifestReader interface {
 }
 
 type wsPipelineWriter interface {
-	WritePipelineBuildspec(marshaler encoding.BinaryMarshaler) (string, error)
-	WritePipelineManifest(marshaler encoding.BinaryMarshaler) (string, error)
+	WritePipelineBuildspec(marshaler encoding.BinaryMarshaler, name string) (string, error)
+	WritePipelineManifest(marshaler encoding.BinaryMarshaler, name string) (string, error)
```

Comment:
super nit: what do you think of flipping the parameter order just to be kindof consistent to [`os.WriteFile`](https://pkg.go.dev/os#WriteFile)

```suggestion
	WritePipelineBuildspec(name string, data encoding.BinaryMarshaler) (string, error)
	WritePipelineManifest(name string, data encoding.BinaryMarshaler) (string, error)
```

Reply:
normally I'd that sounds like a great change, however since `WriteServiceManifest()` and `WriteJobManifest()` have this ordering, I think it makes more sense to stay consistent with them since they are right next to these functions. 

I would be open to changing all of them, but probably a little out of scope for this PR.
https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/workspace/workspace.go#L313-L328